src/test/java/BaseTest.java

```diff
@@ -32,18 +37,39 @@ public void tearDownBrowser() {
 
     @BeforeMethod
     @Parameters({"baseURL"})
-    public void launchBrowser(String baseURL) {
-//        driver = new ChromeDriver();
-//      driver = new FirefoxDriver();
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-        if(baseURL == null)
+    public void launchBrowser(String baseURL) throws MalformedURLException {
+    if(baseURL == null)
             baseURL = "https://bbb.testpro.io";
-        System.setProperty("webdriver.gecko.driver", "geckodriver");
-        driver = new FirefoxDriver();
+        driver = pickBrowser(System.getProperty("browser"));
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/204729769-c312fbb9-4990-41b0-ac35-e0f85eb5e6a0.png)

Here is the fix:  `driver = pickBrowser("browser");`

![image](https://user-images.githubusercontent.com/104761482/204729987-3e0616e9-d37d-40be-9fd8-0f8a420bce59.png)